tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs

```diff
@@ -76,29 +76,29 @@ public async Task ARLINGTON_ROPC_AAD_Async()
         public async Task ARLINGTON_ROPC_ADFS_Async()
         {
             var labResponse = await LabUserHelper.GetArlingtonADFSUserAsync().ConfigureAwait(false);
-            await RunHappyPathTestAsync(labResponse).ConfigureAwait(false);
+            await RunHappyPathTestAsync(labResponse, isAad: true).ConfigureAwait(false);
         }
 
         [TestMethod]
         public async Task ROPC_ADFSv4Federated_Async()
         {
             var labResponse = await LabUserHelper.GetAdfsUserAsync(FederationProvider.AdfsV4, true).ConfigureAwait(false);
-            await RunHappyPathTestAsync(labResponse).ConfigureAwait(false);
+            await RunHappyPathTestAsync(labResponse, isAad: true).ConfigureAwait(false);
```

Comment:
shouldn't this be isAdfs?

Reply:
Federated ADFS is not ADFS. It's AAD. All these tests are AAD. There is a "direct ADFS" test below, that does not use RunHappyPathTestAsync